### DISCUSSION TITLE
Difficulty sorting criteria works incorrect

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15375] [Q&A] [answered]
So problem is incorrect sorting by a difficulty criteria. This vid shows this problem:

https://user-images.githubusercontent.com/86934170/139557797-990a4ef3-606a-4f8a-8229-aefc370a2ac3.mp4

Sorting by a difficulty is the only criteria that requires to separate difficulties of the beatmaps. For example, how it looks like in osu!stable

![image](https://user-images.githubusercontent.com/86934170/139557845-6c8b313f-427a-405a-b1b0-4ceca7530590.png)

This is exactly how it should be in a lazer.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/2547
https://github.com/ppy/osu/issues/5390
